domains/dom1.conf

```diff
@@ -0,0 +1,103 @@
+{
+    domain_names = {
+        dom1 = 'Domain 1',
+        ffffm_event = 'Event'
+    },
+    domain_seed = '264272521fc93692f8ae285688e8f825b01a9376626df8600fead6e4fca9e5d6',
+    hide_domain = { 'dom1' },
```

Comment:
```suggestion
    hide_domain = { 'dom1','ffffm_event' },
```

Reply:
Should be done in https://chaos.expert/FFFFM/domain-builder